src/server/config.ini

```
@@ -236,20 +236,20 @@ stateful = true
   delay_for_fd_timeout_on_start = false
   empty_write_disabled = false
 
-  prepare_timeout_ms_for_secondaries = 3000
-  prepare_timeout_ms_for_potential_secondaries = 5000
+  prepare_timeout_ms_for_secondaries = 1000
+  prepare_timeout_ms_for_potential_secondaries = 3000
```

Based on the experiences of production environment, sometimes both of them would be increased to 6000/10000. Thus I think 1000/3000 would be a litter small, which would make partitions unstable.
```suggestion
  prepare_timeout_ms_for_secondaries = 3000
  prepare_timeout_ms_for_potential_secondaries = 5000
```

---

OK, I've reverted this change and updated the default values in code.